### PR TITLE
Adding test for text-autospace: ideogram-alpha

### PR DESCRIPTION
**Pull request by vitorroriz**

### CODE REVIEW

**Review by nt1m — APPROVED** [review]

File: css/css-text/text-autospace/text-autospace-ideogram-alpha-001.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="UTF-8">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<link rel="help" href="https://drafts.csswg.org/css-text-4/#text-autospace-property">
+<link rel="match" href="text-autospace-ideogram-alpha-001-ref.html">
+<link rel="author" title="Vitor Roriz" href="https://github.com/vitorroriz">
+<style>
+    @font-face {
```
Comment (kojiishi) [inline css/css-text/text-autospace/text-autospace-ideogram-alpha-001.html:10]: nit: `<link rel="stylesheet" herf="/fonts/ahem.css">` is preferred.

File: css/css-text/text-autospace/text-autospace-ideogram-alpha-001.html [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    p {
+        font-size: 40px;
+        margin: 2px;
+        width: fit-content;
+        font-family: ahem;
+    }
+</style>
+</head>
+<body>
+    <p class="no-autospace">A水</p>
```
Comment (kojiishi) [inline css/css-text/text-autospace/text-autospace-ideogram-alpha-001.html:33]: nit: How about adding "1水" to test that the `ideograph-alpha` doesn't insert the spacing to digits?

**Review by kojiishi — COMMENTED** [review]
s/ideogram/ideograph/ in the title.

File: css/css-text/text-autospace/text-autospace-ideogram-alpha-001.html (hunk 1 above)
Comment (nt1m) [reply css/css-text/text-autospace/text-autospace-ideogram-alpha-001.html:10]: (same in the -ref.html file fwiw, the linter is complaining about it)

Comment (vitorroriz) [reply css/css-text/text-autospace/text-autospace-ideogram-alpha-001.html]: Fixed

**Pull request merged**